### PR TITLE
[INLONG-649][Doc] Add the usage document for Apache Hudi

### LINKED ISSUE #649: [Feature][Doc] Add the usage document for Apache Hudi
### Description

_No response_

### Use case

_No response_

### Are you willing to submit PR?

- [X] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by ZuoFengZhang**
### Prepare a Pull Request
*(Change the title refer to the following example)*

[INLONG-649][Doc] Add the usage document for Apache Hudi

- Fixes #649 

### Motivation

*Explain here the context, and why you're making that change. What is the problem you're trying to solve.*

### Modifications

*Describe the modifications you've done.*

### Verifying this change

- [x] Make sure that the change passes the CI checks.

*(Please pick either of the following options)*

This change is a trivial rework / code cleanup without any test coverage.

*(or)*

This change is already covered by existing tests, such as *(please describe tests)*.

*(or)*

This change added tests and can be verified as follows:

*(example:)*
  - *Added integration tests for end-to-end deployment with large payloads (10MB)*
  - *Extended integration test for recovery after broker failure*

### Documentation

  - Does this pull request introduce a new feature? (yes / no)
  - If yes, how is the feature documented? (not applicable / docs / JavaDocs / not documented)
  - If a feature is not applicable for documentation, explain why?
  - If a feature is not documented yet in this PR, please create a followup issue for adding the documentation

### CODE REVIEW

**Review by XuQianJin-Stars — APPROVED** [review]

**Comment by dockerzhang** [comment]
@ZuoFengZhang please add the English version, thanks.

**Review by yunqingmoswu — APPROVED** [review]

File: i18n/zh-CN/docusaurus-plugin-content-docs/current/data_node/load_node/hudi.md [hunk 1]
```diff
@@ -0,0 +1,204 @@
...
+| hoodie.table.name                           | 可选 | String | 将用于向 Hive 注册的表名。 需要在运行中保持一致。            |
+| hoodie.datasource.write.recordkey.field     | 必填 | String | 记录的主键字段。 用作“HoodieKey”的“recordKey”组件的值。 实际值将通过在字段值上调用 .toString() 来获得。 可以使用点符号指定嵌套字段，例如：`a.b.c` |
+| hoodie.datasource.write.partitionpath.field | 可选 | String | 分区路径字段。 在 HoodieKey 的 partitionPath 组件中使用的值。 通过调用 .toString() 获得的实际值 |
+| inlong.metric.labels                        | 可选 | String | 在long metric label中，value的格式为groupId=xxgroup&streamId=xxstream&nodeId=xxnode。 |
+
+
+
+## 数据类型映射
+
+<div class="wy-table-responsive">
```
Comment (dockerzhang) [inline i18n/zh-CN/docusaurus-plugin-content-docs/current/data_node/load_node/hudi.md:125]: please use markdown table

File: i18n/zh-CN/docusaurus-plugin-content-docs-community/current/how-to-report-issues.md [hunk 2]
```diff
@@ -32,16 +32,16 @@ Apache InLong 项目使用 GitHub Issues 来跟踪所有问题。 这些包括
...
+|        组件        |                                  描述                                   |
+|:----------------:|:---------------------------------------------------------------------:|
+|   InLong Agent   |           数据采集 Agent，支持从指定目录或文件读取常规日志、逐条上报。后续也将扩展 DB 采集等能力。           |
+| InLong Dataproxy |  一个基于 Flume-ng 的 Proxy 组件，支持数据发送阻塞和落盘重发，拥有将接收到的数据转发到不同 MQ（消息队列）的能力。   |
+|  InLong TubeMQ   |       腾讯自研的消息队列服务，专注于大数据场景下海量数据的高性能存储和传输，在海量实践和低成本方面有着良好的核心优势。        |
+|   InLong Sort    | 对从不同的 MQ 消费到的数据进行 ETL 处理，然后汇聚并写入 Hive、ClickHouse、Hbase、Iceberg、Hudi 等存储系统。 |
+|  InLong Manager  |                 提供完整的数据服务管控能力，包括元数据、任务流、权限，OpenAPI等。                  |
+| InLong Dashboard |                  用于管理数据接入的前端页面，简化整个 InLong 管控平台的使用。                   |
+|   InLong Audit   |         对 InLong 系统的 Agent、DataProxy、Sort 模块的入流量、出流量进行实时审计对账。         |
+|    InLong SDK    |                     包括 DataProxy SDK, Sort SDK 等                      |
```
Comment (dockerzhang) [inline i18n/zh-CN/docusaurus-plugin-content-docs-community/current/how-to-report-issues.md:44]: Do we need to change the file?

**Comment by dockerzhang** [comment]
@ZuoFengZhang workflow failed. Please check.

File: i18n/zh-CN/docusaurus-plugin-content-docs-community/current/how-to-report-issues.md (hunk 2 above)
Comment (ZuoFengZhang) [reply i18n/zh-CN/docusaurus-plugin-content-docs-community/current/how-to-report-issues.md]: I have rebase this file.

**Comment by dockerzhang** [comment]
related to https://github.com/apache/inlong/issues/6787

**Review by dockerzhang — APPROVED** [review]

**Pull request merged**